src/clr-angular/utils/host-wrapping/host-wrapping.module.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {NgModule} from "@angular/core";
+import {BrowserAnimationsModule} from "@angular/platform-browser/animations";
+
+import {EmptyAnchor} from "./empty-anchor";
+
+
+/**
+ * Internal module, please do not export!
+ */
+// We don't use animations, but the BrowserAnimationsModule introduces inconsistent behavior absolutely everywhere
+// depending on whether it's imported in some distant modue or not. So to be consistent, we include it everywhere.
+// I'm not happy about this hack.
+@NgModule({
+    imports: [BrowserAnimationsModule],
```

Comment:
Did you try the `NoopAnimationsModule`?

Reply:
It would include the `NoopAnimationsModule` in actual production applications, which is kind of messed up when they already import the `BrowserAnimationsModule` too. So I went with the heavier, but real-life one. I have zero confidence in non-prod modules from the animation part, they simply don't care if they break everything.